src/node/rpc/forwarder.h

```diff
@@ -63,22 +64,23 @@ namespace ccf
       auto data_ = plain.data();
       auto size_ = plain.size();
       serialized::write(data_, size_, caller_id);
-      serialized::write(data_, size_, rpc_ctx.client_session_id);
+      serialized::write(data_, size_, rpc_ctx.session.client_session_id);
       serialized::write(data_, size_, rpc_ctx.actor);
+      serialized::write(data_, size_, rpc_ctx.method);
       serialized::write(data_, size_, include_caller);
       if (include_caller)
       {
         serialized::write(data_, size_, caller_cert.size());
         serialized::write(data_, size_, caller_cert.data(), caller_cert.size());
       }
-      serialized::write(data_, size_, data.data(), data.size());
+      serialized::write(data_, size_, rpc_ctx.raw.data(), rpc_ctx.raw.size());
```

Comment:
This feels like a half-finished bodge that works by chance. I _think_ we should either be sending everything in the context _apart_ from the original raw data, or _only_ the raw data.

Reply:
Please ticket that.

Reply:
We'll have to keep forwarding `caller_id`, `rpc_ctx.session.client_session_id` and, optionally, `caller_cert` as these are not part of the raw data?

Reply:
Will deal with this in #511 when we can look at it in isolation.